osbs/utils.py

```diff
@@ -195,14 +195,14 @@ def checkout_git_repo(git_uri, git_ref, git_branch=None):
 
 
 @contextlib.contextmanager
-def paused_builds(osbs):
+def paused_builds(osbs, quota_name=None):
     logger.info("pausing builds")
-    osbs.pause_builds()
+    osbs.pause_builds(quota_name=quota_name)
```

Comment:
I'm wondering if we should move the `osbs.pause_builds()` call inside the `try` block. We can `resume_builds()` without having to have paused them in the first place without error, and the pause call will put the resource quota in place and then wait for builds to finish.

If we interrupt it during waiting, we'll leave the resource quota in place. I think that's what must have happened when we saw this acting up previously.